aag/weather.py

```diff
@@ -768,7 +770,15 @@ def make_safety_decision(self, current_values):
 
         rain = self._get_rain_safety(current_values)
 
-        safe = cloud[1] & wind[1] & gust[1] & rain[1]
+        saftey_params = {'cloud': cloud[1], 'wind': wind[1], 'gust': gust[1], 'rain': rain[1]}
+
+        if ignore:
+            for weather_to_ignore in ignore:
+                if weather_to_ignore in saftey_params:
+                    del saftey_params[weather_to_ignore]
+        else:
+            safe = all(saftey_params.values())
```

Comment:
Assuming the `None` above, you can listify it with our other tools. Also need to remove the `else` statement as you always want to assign to `safe`.  I'm also logging the ignored value, so pop it rather than delete it so we have access to it after. If this will be too noisy could leave off.

```suggestion
        from panoptes.utils.utils import listify  # at top of file
        
        if ignore is not None:
            for weather_to_ignore in listify(ignore):
                ignored_value = safety_params.pop(weather_to_ignore)

                # Warn if ignoring an unsafe value.
                if ignored_value is False:
                    logger.warning(f'Ignored unsafe value: {weather_to_ignore}={ignored_value}')
                
        # Do final safety check.
        safe = all(safety_params.values())
```